pom.xml

```diff
@@ -1101,5 +1101,61 @@
         </plugins>
       </build>
     </profile>
+    <!-- Profiles preconfiguring different Jenkins versions. -->
+    <!-- LTS lines are used. -->
+    <profile>
+      <id>jenkins-642</id>
+      <properties>
+        <jenkins.version>1.642.1</jenkins.version>
+        <hpi-plugin.version>1.115</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>7</java.level>
+      </properties>
+    </profile>
+    <profile>
+      <id>jenkins-625</id>
+      <properties>
+        <jenkins.version>1.625.3</jenkins.version>
```

Comment:
Once you have this, why do not remove the top level `<properties>` and activate this profile by default?